command/cp.go

```diff
@@ -150,11 +164,12 @@ var copyCommand = &cli.Command{
 			flatten:          c.Bool("flatten"),
 			followSymlinks:   !c.Bool("no-follow-symlinks"),
 			storageClass:     storage.StorageClass(c.String("storage-class")),
-			concurrency:      c.Int("concurrency"),
```

Comment:
These flags are still needed. I guess you removed them by mistake.

Reply:
Oh yeah, thanks for pointing it out. In previous commits, these values were handled using different wrapper type, after removing them I should have restored these.